content/news/023/index.md

```diff
@@ -534,17 +529,17 @@ rusty-editor updates:
 
 ![2D Sight Example](lineofsight.png)
 
-[@basstabs] published a tutorial which explains how to write a
+[@basstabs] published a tutorial that explains how to write a
 line of sight algorithm for 2D games in Rust without using trigonometry or
 square roots. It includes vector diagrams and typeset math to explain the ideas
 behind each stage of the algorithm, source code for each step, tests to verify
-accuracy of the methods, and suggestions for further improvements.
+the accuracy of the methods, and suggestions for further improvements.
 Additionally, the
 [repository](https://github.com/basstabs/2d-line-of-sight) contains benchmarks
 and a sample application written in [ggez](https://crates.io/crates/ggez).
 
-_Discussions: [/r/rust_gamedev](
-https://www.reddit.com/r/rust\_gamedev/comments/nx79kq/)_
+_Discussions:
+[/r/rust_gamedev](https://reddit.com/r/rust\_gamedev/comments/nx79kq/)_
```

Comment:
The escape character in `\` is removed by zola ? Because when i try this url in firefox, `\` is replaced by `/` by firefox.

Reply:
yep, it's part of Markdown as far as I know. seems to work fine [in the preview](https://rust-gamedev-preview.netlify.app/news/023/#a-trig-less-line-of-sight-algorithm-for-2d-games)